src/databricks/labs/ucx/framework/logger.py

```diff
@@ -54,6 +54,12 @@ def format(self, record: logging.LogRecord):  # noqa: A003
         return f"{self.GRAY}{ts}{self.RESET} {level} {color_marker}[{name}] {msg}{self.RESET}"
 
 
+class FileFormatter(logging.Formatter):
```

Comment:
Just initialize it in tasks.py, there's no logic in this formatter for it to have a dedicated class - just configuration. Also display seconds for the log in file.

Reply:
Done